content/ops_guides/ha_guide.md

```diff
@@ -48,34 +48,35 @@ There are specific packages that are commonly included in RHEL/CentOS. You must
 
 Cloudify Manager requires at least two network interfaces:
 
-*   Private - This interface dedicated for communication with other Cloudify components, including agents and cluster members.
-*   Public - This interface dedicated for connections to the Cloudify Manager with the Cloudify CLI and Web Interface.
+
+*   **Private** - Dedicated for communication with other Cloudify components, including agents and cluster members.
+*   **Public** - Dedicated for connections to the Cloudify Manager with the Cloudify CLI and Web Interface.
 
 In some cases, it is possible to use only one network interface, but this can lead to security problems.
 
-### Network Ports
+### Network Ports Requirements
 
-Cloudify Manager listens on the following ports:
+**CLoudify Manager single node:**
 
  
-| Port | Description |
-|------|-------------|
-| 80 | REST API and UI. This port must be accessible when SSL is not enabled. |
-| 22 | REST API and UI. This port must be accessible when SSL is enabled. |
-| 443 | For remote access to the manager from the Cloudify CLI. (Optional) |
-| 5671 | RabbitMQ. This port must be accessible from agent VMs. |
-| 53229 | File server. This port must be accessible from agent VMs. |
-| 53333 | Internal REST communications. This port must be accessible from agent VMs. |
-
-Additionally, when the Manager is a part of a Cloudify Manager cluster, the following ports must be accessible from all the other nodes in the cluster
-
-| Port | Description |
-|------|-------------|
-| 8300 | Internal port for the distributed key/value store. |
-| 8301 | Internal port for TCP and UDP heartbeats. Must be accessible for both TCP and UDP. |
-| 8500 | Port used for outage recovery in the event that half of the nodes in the cluster failed. |
-| 15432 | Database replication port. |
-| 22000 | Filesystem replication port. |
+| Source | <-> | Target | Port | Description |
+|--------|-----------|--------|------|-------------|
+| CLI Client | -> | Cloudify Manager | 80 | REST API and UI. This port must be accessible when SSL is not enabled. |
+| CLI Client | -> | Cloudify Manager | 22 | REST API and UI. This port must be accessible when SSL is enabled. |
+| CLI Client | -> | Cloudify Manager | 443 | For remote access to the manager from the Cloudify CLI. (Optional) |
+| CLI Client | -> | Cloudify Manager | 5671 | RabbitMQ. This port must be accessible from agent VMs. |
```

Comment:
@barucoh 5671 and 53333 are actually agent->manager, not the CLI (cluster or not); 53229 is not used and the old docs were a lie :(

Reply:
@tehasdf right, I'll fix that.
Regarding 53229 however, I can see on both managers (4.4), that are part of a cluster (doesn't necessarily mean it's a cluster port), that they are listening on 53229.